attachment_swift/models/ir_attachment.py

```diff
@@ -85,22 +85,22 @@ def _get_swift_connection(self):
         host = os.environ.get('SWIFT_AUTH_URL')
         account = os.environ.get('SWIFT_ACCOUNT')
         password = os.environ.get('SWIFT_PASSWORD')
-        tenant_name = os.environ.get('SWIFT_TENANT_NAME')
+        project_name = os.environ.get('SWIFT_PROJECT_NAME')
```

Comment:
To ease the transition and prevent some mistakes on deployments, I would still support the old name `SWIFT_TENANT_NAME`, with a warning. Example:

```python
project_name = os.environ.get('SWIFT_PROJECT_NAME')
if not project_name and os.environ.get('SWIFT_TENANT_NAME'):
    project_name = os.environ['SWIFT_TENANT_NAME']
    _logger.warning(
        "SWIFT_TENANT_NAME is deprecated and "
        "must be replaced by SWIFT_PROJECT_NAME"
    )
```